internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -465,7 +465,8 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		if !cd.rendered {
 			continue
 		}
-		if err := r.client.Apply(ctx, cd.resource, resource.MustBeControllableBy(cr.GetUID())); err != nil {
+		if err := r.client.Apply(ctx, cd.resource,
+			append(mergeOptions(tas), resource.MustBeControllableBy(cr.GetUID()))...); err != nil {
```

Comment:
Style nit (feel free to ignore) - personally I'd prefer the longer line here than wrapping like this, because wrapping makes it a bit harder to visually distinguish the wrapped argument to `Apply` from the start of the nested block.

Reply:
Done.